docs/en/setup/backend/backend-receivers.md

```diff
@@ -148,6 +148,22 @@ receiver-meter:
   default:
 ```
 
+To activate the meter rule files:
+```yaml
+agent-analyzer:
+  selector: ${SW_AGENT_ANALYZER:default}
+  default:
+    meterAnalyzerActiveFiles: ${SW_METER_ANALYZER_ACTIVE_FILES:} # Which files could be meter analyzed, files split by ","
+```
+
+The receiver adds labels with `key = service` and `key = instance` to the collected data samples,
```

Comment:
Is meter system doing this?

Reply:
The agent analyzer analyzes the protocol and adds these two keys as labels. After this, the `MAL` could build the entity from these labels.